### DISCUSSION TITLE
᭙ᦔꪀꪑᦔ

### DISCUSSION THREAD

**Discussion by Lowabee** [share121/inter-knot#5214] [灌水]
在绳网能找到灵魂伴侣吗？一个人下空洞太孤独了

**Comment by voidomaino** [comment]
灵魂伴侣还是在现实中找吧，网络环境太虚假，千万警惕诈骗！

**Comment by HLXS-HLXS** [comment]
十个网恋九个惨，还有一个特别惨。<img src="https://bbs-static.miyoushe.com/static/2023/11/16/9869e04ba6a0d0e6ed2418efc27d4b74_4411752698018709526.png" alt="苍角-诶？" width="75" >

**Comment by Kazuha-hhh** [comment]
不建议，网上鱼龙混杂，你懂的
